src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheet with a `.scss` extension shouldn't be linked directly in HTML. You should compile SCSS to CSS and link the CSS file instead.